wordpress-mobile/WordPress-Android
# Issue/7996 actionable empty states (Part 1)

### Fix
Add the actionable empty state view class, [`ActionableEmptyView`](), and implement a single instance of the actionable empty states described in https://github.com/wordpress-mobile/WordPress-Android/issues/7996.  Only one instance is included in this pull request to show the new class in action and keep the changeset small.  The remaining actionable empty states  in https://github.com/wordpress-mobile/WordPress-Android/issues/7996 will be included in a subsequent pull request.  See the screenshots below for illustration.

![aes_media](https://user-images.githubusercontent.com/3827611/42861444-d9bd2aec-8a0f-11e8-818e-d0bff023f42d.png)

### Test
1. Go to ***My Site*** tab.
2. Tap ***Media*** item under ***Publish*** section.
3. Tap ***All***, ***Images***, ***Documents***, ***Videos***, and ***Audio*** tabs.
4. Notice empty view when no media is present.
5. Tap ***Upload Media*** button.
6. Notice new media popup menu.
